src/EnergyPlus/SQLiteProcedures.hh

```diff
@@ -56,10 +56,10 @@ public:
 	// This allows for testing of private methods in SQLite
 	friend class SQLiteFixture;
 
-	void addScheduleData( int const number, std::string const name, std::string const type, double const minValue, double const maxValue );
+	void addScheduleData( int const number, std::string const & name, std::string const & type, double const minValue, double const maxValue );
```

Comment:
@DeadParrot The sqlite code in EnergyPlus is not multithreaded. I ran across this issue about 6 months ago so my memory is a little foggy about the specifics. 

If I am remembering correctly I think this was specific to unit tests and not EnergyPlus runs... Yes that seems right [based on my comments here](https://github.com/NREL/EnergyPlus/commit/96ab8b33e256a3e2a908db91083cc407a2c1768b) and [more specifically the changes here](https://github.com/NREL/EnergyPlus/commit/96ab8b33e256a3e2a908db91083cc407a2c1768b#diff-af56b6d25d63e1535f49f1d4cd8926d8R792).

It appears I fixed the issue in the unit tests themselves but forgot to change back to pass by reference. So if the unit tests pass on all platforms with this change then it seems good from my end. I may double check a full annual run SQL output but I imagine that will be fine.